packages/plugin-typescript/test/cases/develop.default/develop.default.spec.js

```diff
@@ -58,37 +58,23 @@ describe('Develop Greenwood With: ', function() {
 
     describe('Develop command specific .ts behaviors', function() {
       let response = {};
+      let data;
 
       before(async function() {
-        return new Promise((resolve, reject) => {
-          request.get({
-            url: `http://127.0.0.1:${port}/main.ts`
-          }, (err, res) => {
-            if (err) {
-              reject();
-            }
-
-            response = res;
-
-            resolve();
-          });
-        });
+        response = await fetch(`http://127.0.0.1:${port}/main.ts`);
+        data = await response.text();
       });
 
-      it('should return a 200', function(done) {
-        expect(response.statusCode).to.equal(200);
-
-        done();
+      it('should return a 200', function() {
+        expect(response.status).to.equal(200);
```

Comment:
@thescientist13 
Just wanted to double check on these changes before I start to apply them to the other files. I only really ask because it doesn't seem like anything broke when I changed the spec.

Reply:
yeah, this looks to be on the right track!  I feel like there was a case I tried that was tricky and couldn't get it work.  Maybe do a "spot" test on the `request.post` usages in [this spec](https://github.com/ProjectEvergreen/greenwood/blob/release/0.29.0/packages/cli/test/cases/develop.default/develop.default.spec.js) on the release/0.29.0 branch?

Also worth testing in any [`serve` based test case](https://github.com/ProjectEvergreen/greenwood/blob/release/0.29.0/packages/cli/test/cases/serve.default/serve.default.spec.js), I noticed in those cases I had to use `.127.0.0.1` and maybe that was where I got stuck?

But if you can get that cross section of test cases working, then I think it should be full steam ahead at that point.

Reply:
@thescientist13 

> Maybe do a "spot" test....
Was successful on the spot test: https://github.com/ProjectEvergreen/greenwood/compare/release/0.29.0...DevLab2425:greenwood:issue-1016-release-update

> I had to use .127.0.0.1 .....
Will report back on the IP-based tests.

Reply:
I was also successful with replacing the IP with localhost.

Updated compare: https://github.com/ProjectEvergreen/greenwood/compare/release/0.29.0...DevLab2425:greenwood:issue-1016-release-update

Reply:
Do you want me to target the v0.29.0 branch for this PR too? It'll have ~90 updates/~25 files with all of the replacements.

Reply:
Yeah, if you've got it all figured out, feel free to target that release branch! 🙏